torch/distributed/tensor/parallel/style.py

```diff
@@ -320,6 +277,28 @@ def make_output_tensor(
     ).to_local()  # type: ignore[call-arg]
 
 
+@_prepare_output_validate  # type: ignore[arg-type] # pyre-ignore[56]
+def make_sharded_output_tensor(
+    output: DTensor, _device_mesh: Optional[DeviceMesh] = None
```

Comment:
is there a reason we pass in `_device_mesh`? I thought we don't need this to convert a DTensor to torch.Tensor?

Reply:
right, but we do explicitly pass in here:
https://github.com/pytorch/pytorch/blob/main/torch/distributed/_tensor/api.py#L531